CMake/Modules/Urho3D-CMake-common.cmake

```diff
@@ -93,6 +93,20 @@ cmake_dependent_option (URHO3D_SSE "Enable SSE instruction set" ${URHO3D_DEFAULT
 if (CMAKE_PROJECT_NAME STREQUAL Urho3D)
     cmake_dependent_option (URHO3D_LUAJIT_AMALG "Enable LuaJIT amalgamated build (LuaJIT only)" FALSE "URHO3D_LUAJIT" FALSE)
     cmake_dependent_option (URHO3D_SAFE_LUA "Enable Lua C++ wrapper safety checks (Lua/LuaJIT only)" FALSE "URHO3D_LUA OR URHO3D_LUAJIT" FALSE)
+
+    set(LUA_RAW_DESC "Prefer loading raw script files from the file system before falling back on Urho3D resource cache. Useful for debugging (e.g. breakpoints), but less performant (Lua/LuaJIT only)")

```

Comment:
I think all these code can be made much simpler. I would just write it as:
```
if (CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_CONFIGURATION_TYPES)
    set (URHO3D_DEFAULT_LUA_RAW FALSE)
else ()
    set (URHO3D_DEFAULT_LUA_RAW TRUE)
endif ()
cmake_dependent_option (URHO3D_LUA_RAW_SCRIPT_LOADER "Prefer loading raw script files from the file system before falling back on Urho3D resource cache. Useful for debugging (e.g. breakpoints), but less performant (Lua/LuaJIT only)" ${URHO3D_DEFAULT_LUA_RAW} "URHO3D_LUA OR URHO3D_LUAJIT" FALSE)
```

Things to note:

- In CMake, we have an unwritten convention: use double quotes for string literal only when it is absolutely necessary. Don't quote "Release".
- CMAKE_BUILD_TYPE variable is only used by single-configuration generator, such as Eclipse CDT4. So, I have set the default to false also for the multi-configurations generator.